src/SFA.DAS.FAT.Application.UnitTests/Courses/Queries/GetCourses/WhenGettingCourses.cs

```diff
@@ -1,143 +1,94 @@
-        }
-
-        [Test, MoqAutoData]
-        public async Task Then_The_Service_Is_Called_Passing_Keywords_From_Query(
-            GetCoursesQuery request,
-            TrainingCourses courseResponse,
-            [Frozen] Mock<ICourseService> mockService,
-            GetCoursesQueryHandler handler)
-        {
-            //Arrange
-            request.RouteIds = null;
-            request.Levels = null;
-            mockService.Setup(x => x.GetCourses(request.Keyword, null, null, OrderBy.None, request.ShortlistUserId)).ReturnsAsync(courseResponse);
-
-            //Act
-            var actual = await handler.Handle(request, CancellationToken.None);
-
-            //Assert
-            mockService.Verify(x => x.GetCourses(request.Keyword, null, null, OrderBy.None, request.ShortlistUserId), Times.Once);
-            actual.Should().NotBeNull();
-            actual.Courses.Should().BeEquivalentTo(courseResponse.Courses);
-            actual.Sectors.Should().BeEquivalentTo(courseResponse.Sectors);
-            actual.TotalFiltered.Should().Be(courseResponse.TotalFiltered);
-            actual.Total.Should().Be(courseResponse.Total);
-        }
-
-        [Test, MoqAutoData]
-        public async Task Then_Keywords_And_Sectors_Are_Passed_To_The_Service_And_Levels_Is_Null(
-            GetCoursesQuery request,
-            TrainingCourses courseResponse,
-            [Frozen] Mock<ICourseService> mockService,
-            GetCoursesQueryHandler handler)
-        {
-            //Arrange
-            request.Levels = null;
-            mockService.Setup(x => x.GetCourses(request.Keyword, request.RouteIds, null, OrderBy.None, request.ShortlistUserId)).ReturnsAsync(courseResponse);
-
-            //Act
-            var actual = await handler.Handle(request, CancellationToken.None);
-
-            //Assert
-            mockService.Verify(x => x.GetCourses(request.Keyword, request.RouteIds, null, OrderBy.None, request.ShortlistUserId), Times.Once);
-            actual.Should().NotBeNull();
-        }
+namespace SFA.DAS.FAT.Application.UnitTests.Courses.Queries.GetCourses;
 
-        [Test, MoqAutoData]
-        public async Task Then_Keywords_And_Levels_Are_Passed_To_The_Service_And_Sectors_Is_Null(
-            GetCoursesQuery request,
-            TrainingCourses courseResponse,
-            [Frozen] Mock<ICourseService> mockService,
-            GetCoursesQueryHandler handler)
-        {
-            //Arrange
-            request.RouteIds = null;
-            mockService.Setup(x => x.GetCourses(request.Keyword, null, request.Levels, OrderBy.None, request.ShortlistUserId)).ReturnsAsync(courseResponse);
-
-            //Act
-            var actual = await handler.Handle(request, CancellationToken.None);
+public class WhenGettingCourses
+{
+    private Mock<ICourseService> _courseServiceMock;
+    private Mock<ILevelsService> _levelsServiceMock;
+    private Mock<IRoutesService> _routesServiceMock;
+    private GetCoursesQueryHandler _handler;
```

Comment:
When using this style of file names, it would be better if we use target's variable name as `sut` or similar to make it apparent